### ISSUE TITLE
Add support for built-in function 'tonumber'

### ISSUE THREAD

**Issue by kenjones-cisco** [mikefarah/yq#71]
Add support for built-in [function 'tonumber'](https://stedolan.github.io/jq/manual/#example40)

```
The tonumber function parses its input as a number. It will convert correctly-formatted strings to their numeric equivalent, leave numbers alone, and give an error on all other input.
```

**Comment by vanaf** [comment]
So is it available?

**Comment by RichardBronosky** [comment]
Is it available? Bueller?

**Comment by mikefarah** [comment]
Nope - sorry I closed a bunch of these that weren't getting attention. Happy to reopen and look at it.

**Comment by mikefarah** [comment]
Fixed in 4.40.3

**Comment by sszemeresky** [comment]
> Fixed in 4.40.3

where can i find an example of how to use it?
